src/main/java/sirius/db/mixing/types/BaseEntityRef.java

```diff
@@ -180,21 +193,55 @@ public E fetchValue() {
         return value;
     }
 
+    /**
+     * Returns the effective entity object which is referenced by forcing a fresh database lookup.
+     *
+     * @return the entity being referenced or <tt>null</tt> if no entity is referenced.
+     */
+    public E forceFetchValue() {
+        value = null;
+        return fetchCachedValue();
```

Comment:
Optional: Would feel a bit natural to bury the find lookup here and to let fetchCachedValue fallback to forceFetchValue insted of the other way around...